src/ir/auth_logic/ast_construction_test.cc

```diff
@@ -73,4 +84,62 @@ INSTANTIATE_TEST_SUITE_P(
              datalog::Predicate("mayB", {R"("NotifierB")"},
                                 datalog::Sign::kNegated)})));
 
+TEST_P(SaysAssertionAstConstructionTest, SimpleTestWithSaysAssertionProgram) {
```

Comment:
I am wondering it is worth splitting this in a separate file `says_assertion_ast_construction_test.cc`?

Reply:
Probably a neat way of testing would be to do end to end testing. From input text input to the parser to DLIR level (The way Andrew implemented in rust version). And have separate files for testing says_assertion, queries, relationDeclarations and various combinations of these programs.Let me know what you think about.

Reply:
Done! Separated query and says_assertion testing into 2 files with ToString functions in a common header file.